src/servers/model/server/api.ts

```diff
@@ -290,9 +292,11 @@ export function normalizeRestCommonParams(params: any): CommonQueryParamsInfo {
     return {
         model_nums: params.model_nums ? ('' + params.model_nums).split(',').map(n => n.trim()).filter(n => !!n).map(n => +n) : void 0,
         data_source: params.data_source,
-        copy_all_categories: Boolean(params.copy_all_categories),
+        // Boolean('false') return true
+        copy_all_categories: params.copy_all_categories === 'true',
```

Comment:
This is a breaking change! 

Some people use it as ``copy_all_categories=1``. 

The ``Boolean('false') => true``  because non-empty string is truthy in JS.

The use case if you don't want the param to be true is not to include it or use empty string as the value.

But I agree this is not very intuitive. Maybe add a function 

```ts
function isTrue(v: any) {
  const b = Boolean(v);
  if (!b) return false;
  if (typeof b === 'string') return b !== '0' && b.toLowerCase() !== 'false';
  return true;
}
```

and use that? (please test it before you push it).